src/game/model/const.js

```diff
@@ -12,6 +12,7 @@ export const FPS = 60;
 // export const IS_HIDPI = window.devicePixelRatio > 1;
 export const IS_HIDPI = true // Set high resolution by default
 export const IS_IOS = /iPad|iPhone|iPod/.test(window.navigator.platform);
-export const IS_MOBILE = /Android/.test(window.navigator.userAgent) || IS_IOS;
+export const IS_ANDROID = /Android/.test(window.navigator.userAgent);
 export const IS_TOUCH_ENABLED = 'ontouchstart' in window;
+export const IS_MOBILE = IS_TOUCH_ENABLED;
```

Comment:
I think to detect a mobile/touch device (which can include **non** iOS tablet devices etc.) it's better to test for 'touch' capabilities instead of relying on the platform/userAgent in this case.